src/ruby/puppet-server-lib/puppet/server/oids.rb

```diff
@@ -0,0 +1,31 @@
+module Puppet::Server::Oids
```

Comment:
@haus I think this whole file is going to go away once https://github.com/puppetlabs/puppet/pull/4827 gets merged (sorry, should have made that clear on the PR)